setup.cfg

```diff
@@ -0,0 +1,60 @@
+[metadata]
+name = probfit
+description = Distribution Fitting/Regression Library
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+url = https://github.com/scikit-hep/probfit
+author = Piti Ongmongkolkul
+author_email = piti118@gmail.com
+license = MIT
+license_file = LICENSE
+classifiers =
+    Development Status :: 4 - Beta
```

Comment:
```suggestion
    Development Status :: 7 - Inactive
```
?

Reply:
We should probably add the standard Scikit-HEP maintainer lines here too.

Reply:
Yep, makes very much sense.